test/types/cptr/c_ptrTo_domain.1.good

```diff
@@ -0,0 +1,5 @@
+$CHPL_HOME/modules/standard/Errors.chpl:nnnn: In function 'compilerError':
+$CHPL_HOME/modules/standard/Errors.chpl:nnnn: warning: compiler diagnostic depth value exceeds call stack depth
```

Comment:
The compiler diagnostic depth messages seem unfortunate, do we know what's going on with that?

Reply:
I'm not sure, and actually wasn't aware of the stack-trace depth parameter on `compilerError` before this. I saw that it had been around since before the `c_addrOf` PR, and looking now, this has been around [for 8 years](https://github.com/chapel-lang/chapel/commit/21b548aeb3d2d3e42080f3787a333717f3747849#diff-5a3811b5f4dc468462520d77f9151b268eb5f82fe2724f874a8fba5a1134e6ffR192-R193). I think it would be fine to change, but was slightly hesitant due to how close the release is.

Reply:
Ah, I see, so it's because of that `2` argument.  And without the 2, we warn at a weird spot?

Reply:
Yeah the 2 apparently controls the depth of stack trace printed by the `compilerError`, I've never seen it used before. Without that argument, the warning just looks like:
```
c_ptrTo_domain.chpl:8: error: c_ptrTo domain type not supported
```
So no stack trace printed at all. That seems fine to me, though.

Reply:
Yeah, I'd just remove it, then